manager/controllers/mockup/datacatalog.go

```diff
@@ -63,19 +58,23 @@ func NewTestCatalog() *DataCatalogDummy {
 	tags := taxonomy.Tags{}
 	tags.Items = map[string]interface{}{"PI": true}
 
-	geo := "theshire" //nolint:goconst
+	geo := "theshire"
 	geoExternal := "neverland"
 
 	// TODO(roee88): some of these are also defined in ifdetails.go
 	var csvFormat taxonomy.DataFormat = "csv"
 	var parquetFormat taxonomy.DataFormat = "parquet"
 	var jsonFormat taxonomy.DataFormat = "json"
+	var dummyResourceName = "xxx"
+	var dummyCredentials = "dummy"
+	var kafkaDeserializer = "io.confluent.kafka.serializers.json.KafkaJsonSchemaDeserializer"
+	var sslTruststore = "xyz123"
```

Comment:
Please keep the same style, i.e. <var_name> := <value> without "var" declaration